src/services/permissions-getter.js

```diff
@@ -167,29 +148,67 @@ class PermissionsGetter {
     }
   }
 
-  static _isPermissionExpired(renderingId, permissionName) {
+  static _isLastRetrieveExpired(lastRetrieveTime) {
+    if (!lastRetrieveTime) return true;
     const currentTime = moment();
-    const lastRetrieve = PermissionsGetter.getLastRetrieveTime(renderingId, permissionName);
+    const elapsedSeconds = currentTime.diff(lastRetrieveTime, 'seconds');
+    return elapsedSeconds >= PermissionsGetter.expirationInSeconds;
+  }
 
-    if (!lastRetrieve) {
-      return true;
+  static _isRegularRetrievalRequired(renderingId) {
+    if (PermissionsGetter.isRolesACLActivated) {
+      const lastRetrieveInCollections = PermissionsGetter._getLastRetrieveTimeInCollections();
+      return PermissionsGetter._isLastRetrieveExpired(lastRetrieveInCollections);
     }
+    const lastRetrieveInRendering = PermissionsGetter._getLastRetrieveTimeInRendering(renderingId);
+    return PermissionsGetter._isLastRetrieveExpired(lastRetrieveInRendering);
+  }
 
-    const elapsedSeconds = currentTime.diff(lastRetrieve, 'seconds');
-    return elapsedSeconds >= PermissionsGetter.expirationInSeconds;
+  static _isRenderingOnlyRetrievalRequired(renderingId, permissionName) {
+    const lastRetrieve = PermissionsGetter._getLastRetrieveTimeInRendering(renderingId);
+    return PermissionsGetter.isRolesACLActivated
+      && permissionName === 'browseEnabled'
+      && PermissionsGetter._isLastRetrieveExpired(lastRetrieve);
   }
 
-  async _retrievePermissions(renderingId) {
+  async _retrievePermissions(renderingId, { renderingOnly = false }) {
+    const queryParams = { renderingId };
+    if (renderingOnly) queryParams.renderingSpecificOnly = true;
+
     return forestServerRequester
-      .perform('/liana/v3/permissions', this.environmentSecret, { renderingId })
+      .perform('/liana/v3/permissions', this.environmentSecret, queryParams)
       .then((responseBody) => {
         PermissionsGetter.isRolesACLActivated = responseBody.meta
           ? responseBody.meta.rolesACLActivated
           : false;
-        PermissionsGetter._setPermissions(renderingId, responseBody.data);
+
+        if (!responseBody.data) return null;
+        if (renderingOnly && responseBody.data.renderings) {
```

Comment:
so if `responseBody.data.renderings` is falsy you will set the global permissions 😅 ...